README.md

```diff
@@ -0,0 +1,57 @@
+# ami-builder-configs
+Configuration files for building various AMIs using ami-builder
+
+Each time ami-builder-config code has a new commit & is merged, a new GitHub release is created and zipped
```

Comment:
Is this via CircleCI? If so, probably worth explicitly stating that and maybe linking to CircleCI job?

Reply:
changed as requested